content/toc-doc.json

```diff
@@ -229,6 +229,17 @@
       }
     ]
   },
+  {
+    "category": "Guides",
+    "slug": "/guides",
+    "items": [
+      {
+        "id": "/guides/tinacms/adding-internationalization-to-tina/overview/",
+        "slug": "/guides/tinacms/adding-internationalization-to-tina/overview/",
+        "title": "Adding Internationalization to Tina"
+      }
```

Comment:
Right now, non of the sub-pages are accessible in the sidebar.
I think you'll want to set the `expandOnSelect` to true, and add the "items" list with all the sub-pages.
See: https://github.com/tinacms/tinacms.org/pull/1226/files#diff-6e6909b12b1fddd800e449f17dc6fec70644386d5a6ffdacba915a017195b298L399